templates/quickstart-finance/get-dependencies.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
```

Comment:
I do not like this as it will not work on Windows. Another good reason to fold finance releases into Daml releases so you can make the libraries part of the template.

Reply:
I can create a windows `.bat` version of this file...?

Reply:
We'll consider this as part of another PR